@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/mass/fn_massUpdate.sqf

```diff
@@ -105,6 +105,7 @@ _heli setCenterOfMass [_curLatCG, 7.12 - _curLongCG, -1.34];
 //systemChat format ["Total Mass = %1 lbs (%2 kg) -- Total Moment = %3 -- Long CG = %4 in -- Lat CG = %5 in", (_curMass * 2.20462) toFixed 1, _curMass toFixed 1, _curLongMom toFixed 3, (_curLongCG * 39.3701) toFixed 1, (_curLatCG * 39.3701) toFixed 1];
 
 if (local _heli) then {
+    //_curMass = 8165;
```

Comment:
This is not required?

Reply:
That allows me to fix the mass and circumvent the mass update function, 8165 is 18,000lbs which is the reference weight for the FM.